### DISCUSSION TITLE
Stuck at Your site is ready to be published at

### DISCUSSION THREAD

**Discussion by neerajadhav** [community/community#23884] [Other Feature Feedback, Questions, & Ideas]
<p>I had a repo named <a href="http://neerajadhav.github.io" rel="noopener nofollow ugc">neerajadhav.github.io</a> which had active gh page. Once I deleted the repo and recreated it. BUT NOW GH PAGE SAYS : Your site is ready to be published at</p>

**Comment by kingthorin** [comment, marked as answer]
<p>Working for me. Likely just a cache/CDN issue.</p>

**Comment by neerajadhav** [comment]
<p>Thank you, it’s working now. Ya it was a cache issue :smile:</p>
